website/src/components/guides/theming/defaults.ts

```diff
@@ -110,7 +110,7 @@ const annotationsDefaults: CompleteTheme['annotations'] = {
 const tooltipDefaults: CompleteTheme['tooltip'] = {
     container: {
         background: '#ffffff',
-        color: extendedTheme.textColor,
```

Comment:
I think it should be `extendedTheme.text.fill`.

Reply:
Done!